Gems/ROS2/Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -0,0 +1,485 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "RobotImporter/URDF/CollidersMaker.h"
+#include "RobotImporter/URDF/PrefabMakerUtils.h"
+#include "RobotImporter/Utils/RobotImporterUtils.h"
+#include "RobotImporter/Utils/SourceAssetsStorage.h"
+#include "RobotImporter/Utils/TypeConversions.h"
+#include <AzCore/Asset/AssetManagerBus.h>
+#include <AzCore/Serialization/Json/JsonUtils.h>
+#include <AzCore/StringFunc/StringFunc.h>
+#include <AzToolsFramework/API/EditorAssetSystemAPI.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <SceneAPI/SceneCore/Containers/Scene.h>
+#include <SceneAPI/SceneCore/Containers/Utilities/Filters.h>
+#include <SceneAPI/SceneCore/DataTypes/Groups/ISceneNodeGroup.h>
+#include <SceneAPI/SceneCore/Events/AssetImportRequest.h>
+#include <SceneAPI/SceneCore/Events/SceneSerializationBus.h>
+#include <SceneAPI/SceneCore/Utilities/SceneGraphSelector.h>
+#include <Source/EditorColliderComponent.h>
+
+namespace ROS2
+{
+    namespace Internal
+    {
+        static const char* collidersMakerLoggingTag = "CollidersMaker";
+
+        AZStd::optional<AZ::IO::Path> GetMeshProductPathFromSourcePath(const AZ::IO::Path& sourcePath)
+        {
+            AZ_TracePrintf(Internal::collidersMakerLoggingTag, "GetMeshProductPathFromSourcePath: %s", sourcePath.c_str());
```

Comment:
This AZ_TracePrintf should have a newline at the end of the format string.
```suggestion
            AZ_TracePrintf(Internal::collidersMakerLoggingTag, "GetMeshProductPathFromSourcePath: %s\n", sourcePath.c_str());
```